src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Transport.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\DecodingExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24Transport extends AbstractTransport
+{
+    protected const HOST = 'api.lox24.eu';
+
+    public const ALLOWED_VOICE_LANGUAGES = ['en', 'de', 'es', 'fr', 'it', 'auto'];
+
+    public function __construct(
+        #[\SensitiveParameter] private readonly string $auth,
+        private readonly string                        $from,
+        private readonly array                         $options = [],
+        ?HttpClientInterface                           $client = null,
+        ?EventDispatcherInterface                      $dispatcher = null
+    ) {
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        $params = [
+            'from' => $this->from,
+            ...$this->options,
+        ];
+
+        $query = $params ? '?'.http_build_query($params) : '';
+
+        return "lox24://{$this->getEndpoint()}$query";
```

Comment:
Please use sprintf like in the other transports

Reply:
done